src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -246,7 +248,17 @@ public void render(Graphics g)
 
     // Get a reference to the current action being built, if one exists.
     Unit currentActor = mapController.getContemplatedActor();
-    XYCoord actorCoord = mapController.getContemplationCoord();
+    AnimState actorAnimState = AnimState.IDLE;
+    XYCoord actorDrawCoord = mapController.getContemplationCoord();
```

Comment:
How did we end up with `getContemplatedActor` and `getContemplationCoord`?

Reply:
I blame me. I like to say function names in my head, and those flow nicer than standardizing.

Reply:
But now they aren't consistent aaaargh. I mean whatever, it works. If we didn't have an IDE with auto-complete I'd object more strongly.